App.js

```diff
@@ -2,17 +2,84 @@ import 'react-native-gesture-handler';
 import { createStackNavigator } from '@react-navigation/stack';
 import { NavigationContainer} from '@react-navigation/native';
 import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Dashboard from './screens/Dashboard';
 
-import Dashboard from './screens/Dashboard'
 
 const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
+
+function DashboardStack() {
+  return (
+    <Stack.Navigator
+      initialRouteName="Dashboard"
+      screenOptions={{
+        headerStyle: { backgroundColor: '#3DD5F4' },
+        headerTintColor: '#fff',
+        headerTitleStyle: { fontWeight: 'bold' },
+      }}>
+      <Stack.Screen
+        name="Dashboard"
+        component={Dashboard}
+        options={{ title: 'CoSava Dashboard' }}
+      />
+    </Stack.Navigator>
+  );
+}
+
+/*function SettingsStack() {
+  return (
+    <Stack.Navigator
+      initialRouteName="Settings"
+      screenOptions={{
+        headerStyle: { backgroundColor: '#3DD5F4' },
+        headerTintColor: '#fff',
+        headerTitleStyle: { fontWeight: 'bold' },
+      }}>
+      <Stack.Screen
+        name="Settings"
+        component={SettingsScreen}
+        options={{ title: 'Setting Page' }}
+      />
+    </Stack.Navigator>
+  );
+}*/
 
 export default function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator>
-        <Stack.Screen name="Dashboard" component={Dashboard} />
-      </Stack.Navigator>
+      <Tab.Navigator
+        initialRouteName="Dashboard"
+        tabBarOptions={{
+          activeTintColor: '#3DD5F4',
+        }}>
+        <Tab.Screen
+          name="Dashboardtack"
+          component={DashboardStack}
+          options={{
+            tabBarLabel: 'Dashboard',
+            tabBarIcon: ({ color, size }) => (
+              <MaterialCommunityIcons name="home" color={color} size={size} />
+            ),
+          }}
+        />
+        <Tab.Screen
+          name="SettingsStack"
+          component={DashboardStack}
```

Comment:
Should this be SettingsStack? or is it Dashboard stack since Settings stack isn't functional yet?

Reply:
this will eventually be setting stack but is Dashboard because SettingStack is not functional yet